### ISSUE TITLE
did you stopped  development since since can not do better than lightGBM pr Xgboost pr catboost?

### ISSUE THREAD

**Issue by Sandy4321** [ogrisel/pygbm#96]
did you stopped  development since since can not do better than lightGBM pr Xgboost pr catboost?

**Comment by NicolasHug** [comment]
Not really, development is stalled here because we have been focusing on the scikit-learn implementation (in Cython, not in Numba), which is about as fast as other libraries in most cases.

**Comment by ogrisel** [comment]
The scope of this project is not to be faster than highly optimized libraries like LightGBM and XGBoost but to be almost as fast while significantly simpler in terms of code (high level Python accelerated by numba vs low level C++ for the other two libraries).

**Comment by ogrisel** [comment]
The Cython implementation in scikit-learn is a bit more verbose that pygbm but does not require to add numba as an extra dependency to scikit-learn for the time being.

**Comment by ogrisel** [comment]
BTW, if you are interested in the numba version (that is pygbm), there are a couple of improvements implemented in the Cython version in scikit-learn that could be ported here:

https://github.com/ogrisel/pygbm/labels/help%20wanted

Feel free to contribute a PR if you are interested.
